### ISSUE TITLE
Cloaking Telescope

### ISSUE THREAD

**Issue by 7ijme** [laytan/cloak.nvim#18]
I have the setting cloak_telescope turned on. However, when I use telescope.builtin.find_files, the values aren't cloaked. This is my config:
```lua
return {
        "laytan/cloak.nvim",
        config = function ()
                require('cloak').setup({
                        enabled = true,
                        cloak_character = '*',
                        -- The applied highlight group (colors) on the cloaking, see `:h highlight`.
                        highlight_group = 'Comment',
                        -- Applies the length of the replacement characters for all matched
                        -- patterns, defaults to the length of the matched pattern.
                        cloak_length = nil, -- Provide a number if you want to hide the true length of the value.
                        -- Wether it should try every pattern to find the best fit or stop after the first.
                        try_all_patterns = true,
                        cloak_telescope = true,
                        patterns = {
                                {
                                        -- Match any file starting with '.env'.
                                        -- This can be a table to match multiple file patterns.
                                        file_pattern = '.env*',
                                        -- Match an equals sign and any character after it.
                                        -- This can also be a table of patterns to cloak,
                                        -- example: cloak_pattern = { ':.+', '-.+' } for yaml files.
                                        cloak_pattern = '=.+',
                                        -- A function, table or string to generate the replacement.
                                        -- The actual replacement will contain the 'cloak_character'
                                        -- where it doesn't cover the original text.
                                        -- If left emtpy the legacy behavior of keeping the first character is retained.
                                        replace = nil,
                                },
                        },
                })
        end
}```

**Comment by jokesper** [comment]
What branch of Telescope are you using?
As the readme states in the comment, the feature required for implementation is not yet in v0.1.x.

**Comment by 7ijme** [comment]
Oops, I thought I didn't have a telescope version specified. Apparently I did. Thanks!

**Comment by jokesper** [comment]
I don't know if it makes sense not specifying the version as the branch exists for a reason (probably).
I hope the commit will be merged into 0.1.x at some point until then using master directly might be an option.
